src/simple-length.js

```diff
@@ -38,7 +38,40 @@
     }
   }
 
-  SimpleLength.prototype = shared.LengthValue.prototype;
+  SimpleLength.prototype = Object.create(shared.LengthValue.prototype);
+
+  SimpleLength.prototype.multiply = function(multiplier) {
+    return new SimpleLength((this.value * multiplier), this.type);
+  };
+
+  SimpleLength.prototype.divide = function(divider) {
+    return new SimpleLength((this.value / divider), this.type);
+  };
+
+  SimpleLength.prototype._addSimpleLengths = function(addedLength){
+    if(this instanceof SimpleLength && addedLength instanceof SimpleLength && this.type === addedLength.type){
+      return new SimpleLength((this.value + addedLength.value), this.type);
+    } else {
+      if(!(this instanceof SimpleLength && addedLength instanceof SimpleLength)){
+        throw new TypeError('Objects not of type simple length');
+      } else {
+        throw new TypeError('SimpleLength units are not the same');
+      }
+    } 
```

Comment:
The usual ordering for this is

if (error condition) {
// throw stuff
}

// do stuff